resources/functionMap.php

```diff
@@ -150,7 +150,7 @@
 'AMQPQueue::ack' => ['void', 'deliveryTag'=>'int', 'flags='=>'int|null'],
 'AMQPQueue::bind' => ['void', 'exchangeName'=>'string', 'routingKey='=>'string|null', 'arguments='=>'array'],
 'AMQPQueue::cancel' => ['void', 'consumerTag='=>'string'],
-'AMQPQueue::consume' => ['void', 'callback='=>'callable|null', 'flags='=>'int|null', 'consumerTag='=>'string|null'],
+'AMQPQueue::consume' => ['void', 'callback='=>'callable(AMQPEnvelope, AMQPQueue): mixed|null', 'flags='=>'int|null', 'consumerTag='=>'string|null'],
```

Comment:
This is kind of ambigous and I'm not sure how it's resolved. Please move `null` before the callable, or use parentheses to disambiguate.

Reply:
Fair enough, whoops!